packages/ui/components/form/select/Select.tsx

```diff
@@ -62,33 +62,33 @@ export const Select = <
             props.isDisabled && "bg-gray-100",
             props.classNames?.control
           ),
-        singleValue: () => classNames("text-emphasis placeholder:text-muted", props.classNames?.singleValue),
+        singleValue: () => cx("text-emphasis placeholder:text-muted", props.classNames?.singleValue),
         valueContainer: () =>
-          classNames("text-emphasis placeholder:text-muted flex gap-1", props.classNames?.valueContainer),
+          cx("text-emphasis placeholder:text-muted flex gap-1", props.classNames?.valueContainer),
         multiValue: () =>
-          classNames(
+          cx(
             "bg-subtle text-default rounded-md py-1.5 px-2 flex items-center text-sm leading-none",
             props.classNames?.multiValue
           ),
         menu: () =>
-          classNames(
+          cx(
             "rounded-md bg-default text-sm leading-4 text-default mt-1 border border-subtle",
             props.classNames?.menu
           ),
         groupHeading: () => "leading-none text-xs uppercase text-default pl-2.5 pt-4 pb-2",
-        menuList: () => classNames("scroll-bar scrollbar-track-w-20 rounded-md", props.classNames?.menuList),
+        menuList: () => cx("scroll-bar scrollbar-track-w-20 rounded-md", props.classNames?.menuList),
         indicatorsContainer: (state) =>
-          classNames(
+          cx(
             state.selectProps.menuIsOpen
               ? state.isMulti
                 ? "[&>*:last-child]:rotate-180 [&>*:last-child]:transition-transform"
                 : "rotate-180 transition-transform"
               : "text-default" // Woo it adds another SVG here on multi for some reason
           ),
         multiValueRemove: () => "text-default py-auto ml-2",
-        ...props.classNames,
+        ...classNames,
       }}
-      {...props}
```

Comment:
passing classNames to props overrides all styles of classNames above even if wanted to override only one component.